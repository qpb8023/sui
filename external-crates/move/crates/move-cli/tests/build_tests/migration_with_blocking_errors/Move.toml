
# TOML FILE

[package]
name = "A"

# this is a comment
[addresses]
A = "0x1"
