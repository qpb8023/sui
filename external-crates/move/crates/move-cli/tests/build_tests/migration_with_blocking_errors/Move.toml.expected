
# TOML FILE

[package]
name = "A"
edition = "2024.alpha"

# this is a comment
[addresses]
A = "0x1"
